web/src/views/repo/build/BuildWrapper.vue

```diff
@@ -51,7 +51,7 @@
         <div class="flex justify-between gap-x-4 text-gray-500 flex-shrink-0 pb-2 md:p-0 mx-auto md:mr-0">
           <div class="flex space-x-1 items-center flex-shrink-0">
             <Icon name="since" />
-            <span>{{ since }}</span>
+            <span v-tooltip="'Started at ' + started">{{ since }}</span>
```

Comment:
```suggestion
            <span v-tooltip="'Created at ' + started">{{ since }}</span>
```
this is about creation timestamp

Reply:
If it's about the creation timestamp, then why this change https://github.com/woodpecker-ci/woodpecker/pull/756#discussion_r802090254

Reply:
I just did a code-lookup my-selfe was not sure if it's about start or create